### ISSUE TITLE
[Crash] Removing a Block 

### ISSUE THREAD

**Issue by mcenderdragon** [McJty/AquaMunda#22]
```
java.lang.IllegalArgumentException: Cannot get property PropertyDirection{name=facing, clazz=class net.minecraft.util.EnumFacing, values=[north, south, west, east]} as it does not exist in BlockStateContainer{block=minecraft:air, properties=[]}
	at net.minecraft.block.state.BlockStateContainer$StateImplementation.func_177229_b(BlockStateContainer.java:201)
	at mcjty.immcraft.api.helpers.OrientationTools.getOrientationHoriz(OrientationTools.java:28)
	at mcjty.immcraft.api.generic.GenericBlock.getFrontDirection(GenericBlock.java:132)
	at mcjty.immcraft.api.rendering.BlockRenderHelper.rotateFacing(BlockRenderHelper.java:127)
	at mcjty.aquamunda.blocks.desalination.DesalinationTankTESR.render(DesalinationTankTESR.java:36)
	at mcjty.aquamunda.blocks.desalination.DesalinationTankTESR.func_192841_a(DesalinationTankTESR.java:22)
	at net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_192854_a(TileEntityRendererDispatcher.java:155)
	at net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_180546_a(TileEntityRendererDispatcher.java:130)
	at net.minecraft.client.renderer.RenderGlobal.func_180446_a(RenderGlobal.java:705)
	at net.minecraft.client.renderer.EntityRenderer.func_175068_a(EntityRenderer.java:1411)
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1259)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1062)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1117)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:397)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

```

**Comment by Denoknight** [comment]
Same error happened to me when a creeper explosion removed my fresh water setup. I'm using the latest version of Aqua Munda.

**Comment by McJty** [comment]
I will check it out soon

**Comment by McJty** [comment]
Um seems I already fixed this crash but forgot to release an update :-)

**Comment by McJty** [comment]
Fixed in 0.2.1 which will be approved on curse soon
